windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -106,4 +121,4 @@ $ az vm image list --all --location <location> \// <1>
 ----
 --
 
-* There is currently an issue in Windows Server 2019 versions released after version `10.0.17763.1457` where Windows workloads behind a load balancer are unreachable for clusters installed on AWS. The Windows Server 2019 version `10.0.17763.1457` and earlier are recommended to work around this issue; however, these earlier images are no longer available. This image version unavailability prevents the ability to run Windows workloads behind a load balancer on clusters installed on AWS at this time. See this link:https://github.com/microsoft/Windows-Containers/issues/78[Microsoft Windows Containers issue] for more details.
+* There is currently an issue in Windows Server 2019 versions released after version `10.0.17763.1457` where Windows workloads behind a load balancer are unreachable for clusters installed on AWS and Azure. The Windows Server 2019 version `10.0.17763.1457` and earlier are recommended to work around this issue; however, these earlier images are no longer available for AWS. This image version unavailability prevents the ability to run Windows workloads behind a load balancer on clusters installed on AWS at this time. See this link:https://github.com/microsoft/Windows-Containers/issues/78[Microsoft Windows Containers issue] for more details.
```

Comment:
- s/The Windows Server 2019 version `10.0.17763.1457` and earlier are recommended/Windows Server 2019 version `10.0.17763.1457` or earlier is recommended ?
- s/See this link:/See the link: (because there isn't something that follows "this" except for what is embedded, hope that makes sense!)